packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -1416,7 +1416,53 @@ You can change the `issuer_mode` value using the API or the Admin Console. To en
   * The `consent_method` property can be configured in the App Wizard and on the **General** tab in the Admin Console, but cannot be set using the Dynamic Client Registration API.
   * After an app is created, you can't change the `application_type`.
 
+#### Create an app with a Profile object
```

Comment:
I think this is a great spot to add the example on how to add it. Might be better to specify we're creating an OIDC app instead of just saying `Create an app`?

Reply:
I specify that in the wording just below the title.